libs/shared/util-event/project.json

```diff
@@ -17,13 +17,6 @@
         "jestConfig": "libs/shared/util-event/jest.config.ts",
         "passWithNoTests": true
       }
-    },
```

Comment:
@apolkingg8 What's the purpose of removing the Nx command? Ideally it can be used with `affected` to only update events when relevant libs have changed.

Reply:
Sorry, I just changed the plan in #175. The scripts would be mentioned in README and run manually by developers. So in this case, there is no difference between adding it as Nx command or not. I can add all the scripts into Nx if you want, but listing them in `package.json` is more clear for me. Please let me know your decision.

Reply:
If it's done in `package.json`, how can we implement them as `affected` commands? This will be useful in the future, at least for CI checks.

If you really want them in `package.json`, just keep the Nx command and add `pnpx run <command>` as a script.

Reply:
Ok, I'll migrate all the scripts into Nx project.